data/110/849/700/5/1108497005.geojson

```diff
@@ -43,7 +49,7 @@
         "Landkreis Taichung"
     ],
     "name:eng_x_preferred":[
-        "Taichung County"
```

Comment:
Should this name be stored as an alt?

Reply:
Yes, it should. See details in other PR comment...